File: 백준/Bronze/11328. Strfry/Strfry.py
import sys
input = sys.stdin.readline

n = int(input())
for _ in range(n):
    a, b = input().split()
    
    sort_a = sorted(a)
    sort_b = sorted(b)

    if sort_a == sort_b:
        print("Possible")
    else:
        print("Impossible")

File: 백준/Bronze/11328. Strfry/Strfry.cc
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main(){
    int n;
    cin >> n;
    
    while (n--){
        string a,b;
        cin >> a >> b;
        
        sort(a.begin(), a.end());
        sort(b.begin(), b.end());
        
        if (a==b){
            cout << "Possible" << endl;
        } else {
            cout << "Impossible" << endl;
        }
    }
    
    return 0;
}